### ISSUE TITLE
No actor is created when uploading xfdf file

### ISSUE THREAD

**Issue by shiveredwolf** [spetzel2020/pc-importer#2]
No actor is created when creating an XFDF file.

**Comment by spetzel2020** [comment]
Hard for me to help you unless there's more detail; if you connect with me
on Discord (spetzel#0103) you can send me the XFDF file.

**Comment by spetzel2020** [comment]
Reproduced with Foundry 0.8.x because of the following error:
```
Uncaught (in promise) Error: Invalid PrototypeTokenData displayName which must be a value in CONST.TOKEN_DISPLAY_MODES
    at PrototypeTokenData._validateField (data.mjs:382)
    at PrototypeTokenData.validate (data.mjs:273)
    at PrototypeTokenData.update (data.mjs:452)
    at Actor5e._preCreate (foundry.js:12356)
    at async Actor5e._preCreate (entity.js:676)
    at async ClientDatabaseBackend._preCreateDocumentArray (foundry.js:8573)
    at async ClientDatabaseBackend._createDocuments (foundry.js:8507)
```

Is occurring because `displayName` needs to be set to `0` not `{default: "0"}`

**Comment by spetzel2020** [comment]
Released in v0.7.0 (https://github.com/spetzel2020/pc-importer/releases/latest)
